pkg/router/router.go

```diff
@@ -218,6 +218,11 @@ func (r *router) DialRoutes(
 	lPK := r.conf.PubKey
 	forwardDesc := routing.NewRouteDescriptor(lPK, rPK, lPort, rPort)
 
+	ok := r.checkIfTransportAvailabel(rPK)
```

Comment:
This check should check if any transports are established for the visor and not if there is a transport between local and remote visor. Routes can be multihop and therefore this check in its current form is overly strict. Please change it and also fix the spelling issue with `Avalailable`.

Reply:
The error consequently would be more appropriately named `ErrNoTransportFound`.

Reply:
I feel like `ErrTransportNotFound` should be renamed to something like `ErrFailedToCreateRoute` which is more informative.